src/config/dynamic-theme-fixes.config

```diff
@@ -2730,6 +2730,24 @@ td.c {
     color: ${white} !important;
 }
 
+================================
+
+login.yahoo.com
+
+CSS
+label[for=persistent]::before {
+    background-color: ${black} !important;
+    filter: invert(1) !important;
```

Comment:
Why not using INVERT?

Reply:
I'll try it, i think i landed here by trial and error w/ the selector, so I'll check that INVERT with that selector works

Reply:
Resolved by ca3eead